src/ClientModel.cpp

```diff
@@ -592,8 +591,18 @@ void ClientModel::executeSequence(vector<TransactionSettings> const& _sequence)
 				if (transaction.functionId.isEmpty() || transaction.functionId == ctrInstance.first)
 				{
 					bytes param = encoder.encodedData();
+					bytes contractCode = m_codeModel->linkLibraries(ctrInstance.first, m_deployedLibrary);
+					eth::LinkerObject object = m_codeModel->contract(ctrInstance.first)->linkerObject();
+					if (!object.linkReferences.empty())
+					{
+						for (auto const& ref: object.linkReferences)
+							emit runFailed(QString::fromStdString(ref.second));
+						emit runFailed(ctrInstance.first + " deployment. Cannot link referenced libraries:");
```

Comment:
there's another place in the code where runFailed is called without return. 
i did not put "return" just to stay consistent. 
But this need to review, i will do that in another pr.